src/zope/interface/tests/test_advice.py

```diff
@@ -268,7 +249,7 @@ class B(type):
         self.assertEqual(self._callFUT((A, B,)), Meta_B)
 
     @_skip_under_py2
-    def test_multiple_in_hierarchy_py3k(self):
+    def test_multiple_in_hierarchy_py3k(self): # pragma: no cover (tox runs coverage on Python 2)
```

Comment:
I wonder if you can add `@_skip_under_py2` to .coveragerc and avoid this repetition.  (Or, even better, merge py2+py3 coverage results.)